From 549597c2e07145d4b3e16d5d45bee0b6d2ee58ef Mon Sep 17 00:00:00 2001
From: Hiroki Yamamoto <Hiroki.A.Yamamoto@sony.com>
Date: Thu, 29 Mar 2018 10:37:49 +0900
Subject: [PATCH 2/6] Remove nfca_version.h

Test: compile
Bug: 76393999
Change-Id: Ic3a34e7028f9e6b01e4230c203c6b9bb0bbdeb65
(cherry picked from commit 4ef0bb964d27066fcb26e7602d6007a1d20dc862)
---
 Android.bp                                    |  1 -
 halimpl/bcm2079x/adaptation/HalAdaptation.cpp |  3 --
 src/nfca_version.c                            | 28 -------------------
 3 files changed, 32 deletions(-)
 delete mode 100644 src/nfca_version.c

diff --git a/Android.bp b/Android.bp
index 1a4e181..9520a41 100644
--- a/Android.bp
+++ b/Android.bp
@@ -10,7 +10,6 @@ cc_library_shared {
         "halimpl/bcm2079x/adaptation/*.c",
         "halimpl/bcm2079x/nfc_nci.c",
         "src/adaptation/CrcChecksum.cpp",
-        "src/nfca_version.c",
     ],
     header_libs: [
         "libhardware_headers",
diff --git a/halimpl/bcm2079x/adaptation/HalAdaptation.cpp b/halimpl/bcm2079x/adaptation/HalAdaptation.cpp
index 1c84426..8d3f94a 100644
--- a/halimpl/bcm2079x/adaptation/HalAdaptation.cpp
+++ b/halimpl/bcm2079x/adaptation/HalAdaptation.cpp
@@ -61,8 +61,6 @@ static void BroadcomHalDataCallback(uint16_t data_len, uint8_t* p_data);
 static bool isColdBoot = true;
 
 extern tNFC_HAL_CFG* p_nfc_hal_cfg;
-extern const uint8_t nfca_version_string[];
-extern const uint8_t nfa_version_string[];
 
 tNFC_HAL_DM_PRE_SET_MEM nfc_hal_pre_set_mem_20795a1[] = {
     {0x0016403c, 0x00000008},
@@ -76,7 +74,6 @@ extern tNFC_HAL_DM_PRE_SET_MEM* p_nfc_hal_dm_pre_set_mem;
 
 int HaiInitializeLibrary(const bcm2079x_dev_t* device) {
   ALOGD("%s: enter", __func__);
-  ALOGE("%s: ver=%s nfa=%s", __func__, nfca_version_string, nfa_version_string);
   int retval = EACCES;
   unsigned long freq = 0;
   unsigned long num = 0;
diff --git a/src/nfca_version.c b/src/nfca_version.c
deleted file mode 100644
index 6c83d7d..0000000
--- a/src/nfca_version.c
+++ /dev/null
@@ -1,28 +0,0 @@
-/******************************************************************************
- *
- *  Copyright (C) 2012 Broadcom Corporation
- *
- *  Licensed under the Apache License, Version 2.0 (the "License");
- *  you may not use this file except in compliance with the License.
- *  You may obtain a copy of the License at:
- *
- *  http://www.apache.org/licenses/LICENSE-2.0
- *
- *  Unless required by applicable law or agreed to in writing, software
- *  distributed under the License is distributed on an "AS IS" BASIS,
- *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *  See the License for the specific language governing permissions and
- *  limitations under the License.
- *
- ******************************************************************************/
-#include "bt_types.h"
-
-const uint8_t nfca_version_string[] = "NFCDROID-AOSP_M_00.01";
-
-/*
-// The following string should be manually updated to contain the
-// label of the NFA version being used (see stack-info.txt).
-//
-// NOTE: IF additional branches are used add a "+" at the end of the string
-*/
-const uint8_t nfa_version_string[] = "NFA_PI_1.03.66+";
-- 
2.17.1

